### ISSUE TITLE
Masks on detection test dataset 

### ISSUE THREAD

**Issue by leevyth** [nicolaihaeni/MinneApple#3]
Hi, 
I'm trying to replicate your detection results, but the masks folder on the test data seems to be missing.  Am I suppose to split the training data for evaluation? for now it just breaks the execution right on loading.

**Comment by nicolaihaeni** [comment]
Hi
We do not provide the test masks. However, we provide an evaluation server, where you can submit your results and you will receive the scores as reported in the paper. You can access the evaluation server here: [https://competitions.codalab.org/competitions/21718](https://competitions.codalab.org/competitions/21718).

Let me know if you have any other questions.

**Comment by leevyth** [comment]
Hello, thanks for your fast reply. So if I try to run your the train_rcnn.py script i have to split the train folder images in order for it to have masks on the validation set? either that or don't run the evalute() function?

**Comment by nicolaihaeni** [comment]
Yes, you are correct. I would suggest that you put aside a small number of images for validation to monitor if the network learns something sensible and catch any overfitting.

**Comment by leevyth** [comment]
Thanks :)
